<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change</class>
 <widget class="QDialog" name="change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>768</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>432</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_edit.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>710</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_btn_confirm_1.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>710</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_btn_cancel_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/GUI/UI_login_btn_quit_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>51</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>51</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>名字</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>51</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>130</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改前用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="userCodeLabel">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>156</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改前密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel_2">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>188</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改后用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="userCodeLabel_2">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>214</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改后密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Normal</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameText_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Normal</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="userCodeText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Normal</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="userCodeText_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>650</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Normal</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <zorder>titleLabel</zorder>
   <zorder>nameLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>userNameText</zorder>
   <zorder>userNameText_2</zorder>
   <zorder>userCodeText</zorder>
   <zorder>userCodeText_2</zorder>
   <zorder>saveButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>QuitButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
